### PR TITLE
TEST-01 bb test

### PR DESCRIPTION
**Pull request by mpaz-redstage**

### CODE REVIEW

File: c.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+tes
+test
```
Comment (mauriciopazpp) [inline c.txt:2]: test

**Comment by mauriciopazpp** [comment]
this is a test

**Comment by mauriciopazpp** [comment]
test

**Comment by mpaz-redstage** [comment]
test2

**Comment by azago-redstage** [comment]
testing

**Pull request merged**